_internal/utils/sourceme.R

```diff
@@ -157,11 +8,15 @@ deploy_all_apps <- function(api_key,
 #' @param username The GitHub username or organization name.
 #' @param repo The name of the repository.
 #' @param ref The reference (commit, branch, or tag) to download from the repository. (default: "HEAD")
+#' @param package_repo The R-Universe mirror where the teal packages would be instealled from.
+#' Main version: https://pharmaverse.r-universe.dev and Release version: https://insightsengineering.r-universe.dev
 #' @param ... Additional arguments passed to `shiny::runApp()`.
 #'
 #' @examples
 #' restore_and_run("app1", username = "insightsengineering", repo = "teal.gallery", ref = "HEAD")
-restore_and_run <- function(app_name, username = "insightsengineering", repo = "teal.gallery", ref = "HEAD", ...) {
+restore_and_run <- function(
+    app_name, username = "insightsengineering", repo = "teal.gallery", ref = "HEAD",
```

Comment:
do we need to have username and repo args?

Reply:
I thought it would be helpful if someone forks the repo. It's not super helpful. I'll replace them with function-scoped variables at the top.